src/v2/view-builder/views/webauthn/EnrollWebauthnView.js

```diff
@@ -0,0 +1,126 @@
+import { _, loc, createCallout, createButton } from 'okta';
+import BaseForm from '../../internals/BaseForm';
+import BaseFactorView from '../shared/BaseFactorView';
+import webauthn from '../../../../util/webauthn';
+import BrowserFeatures from 'util/BrowserFeatures';
+import CryptoUtil from '../../../../util/CryptoUtil';
+
+function getExcludeCredentials (credentials = []) {
+  return credentials.map((credential) => {
+    return {
+      type: 'public-key',
+      id: CryptoUtil.strToBin(credential.id),
+    };
+  });
+}
+
+const Body = BaseForm.extend({
+  title: loc('oie.enroll.webauthn.title', 'login'),
+  className: 'oie-webauthn',
+  modelEvents: {
+    'error': '_stopEnrollment',
+  },
+  getUISchema () {
+    const schema = [];
+    const activationData = this.options.appState.get('currentAuthenticator').contextualData.activationData;
+    // Returning custom array so no input fields are displayed for webauthn
+    if (webauthn.isNewApiAvailable()) {
+      schema.push({
```

Comment:
nit:
instead of pushing multiple Views, you could create one View to contain all those following Views and that Single View to `schema`. Feels clearer but up to you.